test/integration/short/timeout-tests.js

```diff
@@ -7,6 +7,7 @@ var Connection = require('../../../lib/connection');
 var utils = require('../../../lib/utils');
 var types = require('../../../lib/types');
 var errors = require('../../../lib/errors');
+var ExecutionProfile = require('../../../lib/execution-profile').ExecutionProfile;
 
 describe('client read timeouts', function () {
   this.timeout(120000);
```

Comment:
Can we take this opportunity to speed up this test suite: use `before()` / `after()` to create the cluster and use `beforeEach()` to resume the paused nodes.
Reusing the same cluster would break test isolation (single test errors can affect following tests in the suite to fail) but this test suite takes ~8 mins to complete...